components/more-less/more-less.js

```diff
@@ -138,16 +150,19 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 			'd2l-more-less-content': true,
 			'd2l-more-less-transition': this.__transitionAdded
 		};
+
+		// The .force-margin-scroll div is used to force content bottom margin to be included in scrollHeight calculations.
+		// The load and slotchange events can be removed when GAUD-8725-more-less-refactor-resizing is removed
 		return html`
 			<div
 				id="${this.__contentId}"
 				class=${classMap(contentClasses)}
 				style=${styleMap({ maxHeight: `${this.__maxHeight}` })}
 				@focusin="${this.__focusIn}"
 				@focusout="${this.__focusOut}"
-				@load=${this.__reactToChanges}
-				@slotchange=${this.#handleSlotChange}>
-				<slot></slot>
+				@load=${this.__reactToChanges}>
```

Comment:
If we want to remove this when the flag is cleaned up, should we only be adding the listener when the flag is false so that we're for sure testing that in the flag on state?